lib/fs.js

```diff
@@ -6,6 +6,7 @@
 const constants = process.binding('constants').fs;
 const util = require('util');
 const pathModule = require('path');
+const internalFs = require('internal/fs');
```

Comment:
This is unfortunately, if I have understood the situation correctly, a breaking change à la #6413?